homework-4/script.js

```diff
@@ -0,0 +1,23 @@
+import users from "./users.json" assert { type: "json" };
+import { convertDollarsToNumber } from "./helpers.js";
+
+function filterUsersByBalance(users) {
```

Comment:
in this case it's better to use array methods chaining [Chaining of Array Methods in JavaScript
](https://www.geeksforgeeks.org/chaining-of-array-methods-in-javascript)
```suggestion
function filterUsersByBalance(users) {
  const result = users
    .filter(user => convertDollarsToNumber(user.balance) > 2000)
    .map(user => user.phone);
    
   return result;
}
```
also if you see some hardcoded data (2000 in this case), you must understand that this is should be a parameter. So your function should have 2 parameters `filterUsersByBalance(array, minBalance)`